apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
+    "postBuild": [
```

Comment:
There should just be one "rush build" in my opinion. "rush rebuild" should really be "rush build -c". "rebuild or not" should just be a parameter. 

I do not intent to support a full range of events at the very beginning, that is why it is alpha API. It is just post "rush build", not per project.